Droidcon-Boston/app/build.gradle

```diff
@@ -65,10 +65,12 @@ dependencies {
 
     // Firebase cloud messaging
     implementation "com.google.firebase:firebase-messaging:$playServices"
+    implementation "com.google.firebase:firebase-database:$playServices"
 
     // Misc
     implementation "com.jakewharton:butterknife:$butterKnife"
     annotationProcessor "com.jakewharton:butterknife-compiler:$butterKnife"
+    implementation 'com.google.code.gson:gson:2.8.2'
```

Comment:
Would be better to use Moshi in new projects...apparently, it's the next version of gson despite gson still being more popular.

Reply:
it's optional though...just thought I'd mention it...

Reply:
gson working here so should leave it for now I think - minor change later if needed. Plus doc on moshi doesn't say definitively better